docs/src/pages/components/data-grid/rendering/DensitySelectorHiddenGrid.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { DataGrid } from '@material-ui/data-grid';
+import { useDemoData } from '@material-ui/x-grid-data-generator';
+
+export default function DensitySelectorHiddenGrid() {
+  const { data } = useDemoData({
+    dataSet: 'Commodity',
+    rowLength: 4,
+    maxColumns: 6,
+  });
+
+  return (
+    <div style={{ height: 300, width: '100%' }}>
+      <DataGrid {...data} hideToolbar={false} disableDensitySelector />
```

Comment:
` hideToolbar={false} ` is redundant, not sure the effect it will have on ppl looking at the source. But I would ask myself why did they put that.
I think we should remove them and wait until/if/when we do the refactoring. We should not think ahead of ourselves.

Reply:
I'm not sure that it really matters, we talked about deferring this problem to a second pull request in https://github.com/mui-org/material-ui-x/pull/606#issuecomment-739489334.

Reply:
I see, but yeah normally you leave things clean because if you change your plan, and defer this refactoring as other priorities come first, then you will have this weird prop for some time in your documentation.

Reply:
Ok why not

Reply:
However, I don't think that developer will ever see this prop as the plan is to remove it before the next release.

Reply:
Well we should release tomorrow or wednesday if we follow our plan...
Also I would argue that if we do showToolbar, we should consider doing showFooter...

Reply:
> showToolbar, we should consider doing showFooter.

I disagree, I think that https://material-ui.com/guides/api/#property-naming is more important. I also think that we don't need showToolbar not hideToolbar.